### ISSUE TITLE
`DialogButton`'s `InkWell` should also use its `radius` property in `borderRadius`

### ISSUE THREAD

**Issue by utkarsh1097** [RatelHub/rflutter_alert#136]
So that the `Ink` effect follows the outline of the button UI that was rendered by `DialogButton` 

Specifically, 

https://github.com/RatelHub/rflutter_alert/blob/master/lib/src/dialog_button.dart#L53-L77

should be 

```
...
        child: InkWell(
          borderRadius: radius,
          highlightColor: highlightColor ?? Theme.of(context).highlightColor,
          splashColor: splashColor ?? Theme.of(context).splashColor,
          onTap: onPressed,
          child: Center(
            child: child,
          ),
        ),
...
```

**Comment by ibrahimdevs** [comment]
@utkarsh1097 this will be fixed in next version: V2.0.6

**Comment by utkarsh1097** [comment]
Thanks!
I see the fix got merged into master. I think it is okay to mark this issue as closed.
